### PR TITLE
documentation(EJ2-64044): Added enable multiple foreign key columns i…

### PR DESCRIPTION
**Pull request by Soundeswari-Selvam**
…n foreignkey column.

### CODE REVIEW

**Comment by sharmi70** [comment]
rebuild

**Review by MaithiliyK — APPROVED** [review]
changes are fine

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/check-box-selection.md [hunk 1]
```diff
@@ -72,3 +72,31 @@ In checkbox selection, selection can also be done by clicking on rows. This sele
 {% endtabs %}
 {% endif %}
 
+## Prevent specific rows from being selected in checkbox selection
+
+You can prevent specific rows from being selected in the checkbox selection mode by hiding the checkboxes using the [RowDataBound](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_RowDataBound) event. You achieve this by setting the `isSelectable` argument as false in the `rowDataBound` event based on certain conditions as per the needs of the application.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/check-box-selection.md:77]: rowDataBound API casing is wrong

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/check-box-selection.md (hunk 1 above)
Comment (Soundeswari-Selvam) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/check-box-selection.md]: Changes are completed.

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**